### ISSUE TITLE
Interaction in ays tmux required when adding OVC templates

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/lib9#139]
#### Installation information

- jumpscale version: 9.3.0
- operating system: Ubuntu 16.04

#### Detailed description

<img width="797" alt="screen shot 2017-12-18 at 12 02 52" src="https://user-images.githubusercontent.com/13795109/34103182-1fb4815a-e3ec-11e7-8311-87ab1e5fa328.png">

#### Steps to reproduce

When executing the following from remote machine:
```python
...
public_ays_url = "http://{}:{}".format(public_ip_address, "5000")
ays = j.clients.ays.get(public_ays_url)
ays.templates.addTemplates("https://github.com/openvcloud/ays_templates", "master")
```

**Comment by yveskerwyn** [comment]
@rkhamis can this be back ported to 9.2.1, it is causing unnecessary support calls

**Comment by rkhamis** [comment]
@arahmanhamdy fix on 9.3.0 and backport to 9.2.1

**Comment by arahmanhamdy** [comment]
fixed in 96f2a8bc456813a4cc8e79c5fa0a7c5b216a6eb3

**Comment by arahmanhamdy** [comment]
fixed in 96f2a8bc456813a4cc8e79c5fa0a7c5b216a6eb3

**Comment by arahmanhamdy** [comment]
Fixed in https://github.com/Jumpscale/core9/commit/96f2a8bc456813a4cc8e79c5fa0a7c5b216a6eb3
